.github/workflows/verify-code.yaml

```diff
@@ -1,63 +1,61 @@
 ---
-name: image-builder
-
+name: coding-checks
```

Comment:
```suggestion
name: code-and-container-pipeline
```

can we use somethink like above or `operator-build-pipeline`? etc

Reply:
done.